src/integration.ts

```diff
@@ -86,24 +86,39 @@ export default class Integration {
     );
   }
 
+  public async makePaginatedRequest(params, regionFilter) {
+    const results = await this.awsProvider.request(
```

Comment:
do we want a test that verifies it can paginate accordingly?

Reply:
Let me see if I can mock that out.